Doc/library/subprocess.rst

```diff
@@ -466,9 +466,9 @@ functions.
       The *pass_fds* parameter was added.
 
    If *cwd* is not ``None``, the function changes the working directory to
-   *cwd* before executing the child.  In particular, the function looks for
-   *executable* (or for the first item in *args*) relative to *cwd* if the
-   executable path is a relative path.
+   *cwd* before executing the child. *cwd* also accepts a `os.PathLike` object.
```

Comment:
:ref:`os.PathLike` would be a preferable fix here (turning this into a cross reference to the ABC's docs)